extension-requests/local-utils.js

```diff
@@ -1,10 +1,21 @@
+const Status = {
+  APPROVED: 'APPROVED',
+  PENDING: 'PENDING',
+  DENIED: 'DENIED',
+};
 async function getExtensionRequests(query = {}) {
   const url = new URL(`${API_BASE_URL}/extension-requests`);
 
   queryParams = ['assignee', 'status', 'taskId'];
-  queryParams.forEach(
-    (key) => query[key] && url.searchParams.set(key, query[key]),
-  );
+  queryParams.forEach((key) => {
+    if (query[key]) {
+      if (Array.isArray(query[key])) {
+        query[key].forEach((value) => url.searchParams.append(key, value));
+      } else {
+        url.searchParams.append(key, query[key]);
+      }
+    }
+  });
```

Comment:
Hi Ajeya, I want to know that Can you make me clear what are you doing here
I am sending query = {status: pending} - https://github.com/Real-Dev-Squad/website-dashboard/pull/434/files#diff-699a12f3b559d9521daf69ab59f425bfcd272e2cb154c288b0b4517dd9da85e1R38
what is value of query[assignee]??

Reply:
here key can be either 'assignee'/ 'status' / 'taskId', so in this query['assigned'] will be undefined. but since we are iterating over only the available keys query['assigned'] won't be evaluated.